code/game/turfs/simulated/floor/plating/asteroid.dm

```diff
@@ -68,7 +68,15 @@
 			if(W.use_tool(src, user, 40, volume=50))
 				if(!can_dig(user))
 					return TRUE
-				to_chat(user, "<span class='notice'>You dig a hole.</span>")
+
+				if(W.tool_behaviour == TOOL_MINING && prob(2)) //I forgot about better mining tools. Brought down the probability and making the options more likely to be boring
+					to_chat(user, "<span class='warning'>You opened a fissure! A gas is escaping...</span>")
+					icon_state = "fissure"
+					icon_plating = "fissure"
+					environment_type="fissure"
+					initial_gas_mix = pickweight(fissure_gas_mix)
```

Comment:
`fissure_gas_mix` doesn't exist on this obj, this should be failing... hm.

Reply:
Fixed the declaration and moved it to the root of asteroid - more details in other request so I'll go ahead and try to resolve this one